src/app/shared/components/template/components/layout/display-group/display-group.component.html

```diff
@@ -3,7 +3,13 @@
   @if (params.sticky === "top") {
     <div [style.height.px]="sticky.height()"></div>
   }
-  <tmpl-display-group-sticky [position]="params.sticky" #sticky>
+  <tmpl-display-group-sticky
+    [position]="params.sticky"
+    [style.--sticky-background]="
+      params.stickyBackground === 'transparent' ? 'transparent' : 'var(--ion-background-color)'
+    "
```

Comment:
nit(blocking): assigning styles in this way looks quite messy. Given the sticky host still renders the child templated content then the child content should be the place to handle this. I'm not sure if maybe I've missed a use case here?